### PR TITLE
Config changes

### PR DESCRIPTION
**Pull request by justin-tay**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: _data/snippets/v7/async-api-dao-disabled.yml [hunk 1]
```diff
@@ -6,12 +6,6 @@ Spring: |+2
   <br/>
   Update your application.yaml with the additional code below. (Only the relevant portions are included.). You will have to initialize your implementation as a Bean.
 
-  ```yaml
```
Comment (aklish) [inline _data/snippets/v7/async-api-dao-disabled.yml:9]: We should remove the 
'update your application.yaml' text above.

File: _data/snippets/v7/async-api-storageengine-disabled.yml [hunk 2]
```diff
@@ -6,12 +6,6 @@ Spring: |+2
   <br/>
   Update your application.yaml with the additional code below. (Only the relevant portions are included.). You will have to initialize your implementation as a Bean.
 
-  ```yaml
```
Comment (aklish) [inline _data/snippets/v7/async-api-storageengine-disabled.yml:9]: Same comment above.

**Review by aklish — CHANGES_REQUESTED** [review]
Looks good.  two small comments.

File: _data/snippets/v7/async-api-dao-disabled.yml (hunk 1 above)
Comment (justin-tay) [reply _data/snippets/v7/async-api-dao-disabled.yml:9]: Have removed the text.

File: _data/snippets/v7/async-api-storageengine-disabled.yml (hunk 2 above)
Comment (justin-tay) [reply _data/snippets/v7/async-api-storageengine-disabled.yml:9]: Have removed the text.

**Review by aklish — APPROVED** [review]

**Pull request merged**